### PR TITLE
move aws mapper tests into separate directory

### PR DESCRIPTION
**Pull request by PradeepKiruvale**
## Proposed changes

Move aws mapper integration tests into a separate directory

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (general improvements like code refactoring that doesn't explicitly fix a bug or add any new functionality)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
<br/>

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [ ] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: tests/RobotFramework/tests/aws/aws_telemetry.robot [hunk 1]
```diff
@@ -0,0 +1,68 @@
+*** Settings ***
+Documentation       Purpose of this test is to verify that tedge-mapper-aws subscribes the topics that are configured as aws.topics
```
Comment (rina23q) [inline tests/RobotFramework/tests/aws/aws_telemetry.robot:2]: Need to update this!

File: tests/RobotFramework/tests/aws/aws_telemetry.robot [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    Should Have MQTT Messages    aws/td/device:main/e/event-type   message_contains="someone logged-in"    date_from=${timestamp}   minimum=1    maximum=1     
+
+
+Publish child device event to te event topic with event type
+    ${timestamp}=        Get Unix Timestamp  
+    Execute Command    tedge mqtt pub te/device/child///e/event-type '{"text": "someone logged-in"}'
+    Should Have MQTT Messages    aws/td/device:child/e/event-type   message_contains="someone logged-in"    date_from=${timestamp}   minimum=1    maximum=1
+
+
+Publish main device alarm to te event topic with event type
```
Comment (rina23q) [inline tests/RobotFramework/tests/aws/aws_telemetry.robot:47]:
```suggestion
Publish main device alarm to te alarm topic with alarm type
```

File: tests/RobotFramework/tests/aws/aws_telemetry.robot [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    ${timestamp}=        Get Unix Timestamp
+    Execute Command    tedge mqtt pub te/device/main///a/alarm-type '{"severity":"critical","text": "someone logged-in"}'
+    Should Have MQTT Messages    aws/td/device:main/a/alarm-type   message_contains="critical"    date_from=${timestamp}   minimum=1    maximum=1     
+
+
+Publish child device alarm to te event topic with event type
```
Comment (rina23q) [inline tests/RobotFramework/tests/aws/aws_telemetry.robot:53]:
```suggestion
Publish child device alarm to te alarm topic with alarm type
```

**Review by rina23q — COMMENTED** [review]
Looks much better than before! Some texts need to be updated.

File: tests/RobotFramework/tests/aws/aws_telemetry.robot (hunk 1 above)
Comment (PradeepKiruvale) [reply tests/RobotFramework/tests/aws/aws_telemetry.robot:2]: done

File: tests/RobotFramework/tests/aws/aws_telemetry.robot (hunk 2 above)
Comment (PradeepKiruvale) [reply tests/RobotFramework/tests/aws/aws_telemetry.robot:47]: fixed

File: tests/RobotFramework/tests/aws/aws_telemetry.robot (hunk 3 above)
Comment (PradeepKiruvale) [reply tests/RobotFramework/tests/aws/aws_telemetry.robot:53]: fixed

**Review by rina23q — APPROVED** [review]
Approved

**Pull request merged**